internal/utils/whitelist.go

```diff
@@ -0,0 +1,39 @@
+package utils
+
+import (
+	"sort"
+	"strings"
+)
+
+type Whitelist map[string]struct{}
```

Comment:
We define some date structure (even kind of ADT) but we use the name not related to its behavior.  There is nothing about `whitelisting`, just set of strings with contains operation.